tox.ini

```diff
@@ -1,6 +1,12 @@
 [tox]
-envlist = test
-toxworkdir = /var/tmp
+envlist = py27,py3
+toxworkdir = {homedir}/.virtualenvs/docker_utils
```

Comment:
this would force py27 and py3 to use the same directory. Can you update this to follow similar pattern to common-tools? https://github.com/confluentinc/common-tools/blob/master/tox.ini#L24

Reply:
https://tox.readthedocs.io/en/latest/config.html

`toxworkdir` != `envdir`
by default, `envdir={toxworkdir}/{envname}`

Reply:
ah misread. Why do we need to change this? Does it not work if using the default (which use .tox in the same dir as tox.ini). We do this for all other projects without problems. 

I think this might be solving a problem with envdir being too long. I guess this could work too instead of setting different env dirs. Note that by setting envdir, it allows us to reuse the same for some things (style) instead of creating a new one. But using one per env is definitely easier to configure.

Reply:
you cannot use/share a single env for both python 2 and python 3.